theano/tensor/nnet/sigm.py

```diff
@@ -81,12 +81,12 @@ def c_code(self, node, name, inp, out, sub):
             or node.inputs[0].type == scalar.float16
         ):
             return (
-                """%(z)s = %(x)s < -88.0f ? 0.0 : %(x)s > 15.0f ? 1.0f : 1.0f /(1.0f + exp(-%(x)s));"""
+                """%(z)s = 1.0f /(1.0f + exp(-%(x)s));"""
```

Comment:
The style unit test are failing and the fix should be as easy as changing this line to

```
            return """%(z)s = 1.0f /(1.0f + exp(-%(x)s));""" % locals()
```

instead of the multi-line version to be consistent with [black](https://github.com/psf/black).